cli/flox/doc/flox-activate.md

```diff
@@ -10,99 +10,103 @@ flox-activate - activate environments
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] activate [ `<options>` ] [ -- `<command>` [ `<argument>` ] ]
+```
+flox [ <general-options> ] activate
+     [-d=<path> | -r=<owner/name>]
+     [ -t ]
+     [ -- <command> [ <arguments> ] ]
+```
 
 # DESCRIPTION
 
-Sets environment variables and aliases, runs hooks and adds environment
-`bin` directories to your `$PATH`. Can be invoked from an interactive
-terminal to launch a sub-shell, non-interactively to produce
-a series of commands to be sourced by your current `$SHELL`,
-or with a command and arguments to be invoked directly.
+Sets environment variables and aliases,
+runs hooks,
+and adds `bin` directories to your `$PATH`.
 
+Launches an interactive subshell when invoked as `flox activate` from an
+interactive shell.
```

Comment:
question/design (nb): what is flox supposed to do in a _non-interactive_ context?
* fail?
* only run the hook?